visionSamples/barcode-reader/app/src/main/java/com/google/android/gms/samples/vision/barcodereader/BarcodeCaptureActivity.java

```diff
@@ -57,7 +57,7 @@
  * rear facing camera. During detection overlay graphics are drawn to indicate the position,
  * size, and ID of each barcode.
  */
-public final class BarcodeCaptureActivity extends AppCompatActivity {
+public final class BarcodeCaptureActivity extends AppCompatActivity implements BarcodeGraphicTracker.UpdateBarcodeListener {
```

Comment:
I think "BarcodeUpdateListener" is more nature and consistent with "BarcodeCaptureActivity", "BarcodeGraphicTracker".

Reply:
Done.